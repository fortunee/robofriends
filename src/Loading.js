const Loading = () => (
  <div className="vh-100 dt w-100">
    <h1 className="tc f1 dtc v-mid">Loading...</h1>
  </div>
);

export default Loading;
