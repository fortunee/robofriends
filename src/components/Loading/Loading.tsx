import { MemoExoticComponent, memo, FC } from 'react';

const Loading: MemoExoticComponent<FC<{}>> = memo(() => (
  <div className="vh-100 dt w-100">
    <h1 className="tc f1 dtc v-mid">Loading...</h1>
  </div>
));

export default Loading;
