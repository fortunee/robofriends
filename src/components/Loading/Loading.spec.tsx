import { shallow } from 'enzyme';
import React from 'react';

import Loading from './Loading';

it('Should render a Loading component', () => {
  expect(shallow(<Loading />).debug()).toMatchSnapshot();
});
