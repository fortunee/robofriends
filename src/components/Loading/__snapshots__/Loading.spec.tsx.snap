// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Loading component 1`] = `
"<div className=\\"vh-100 dt w-100\\">
  <h1 className=\\"tc f1 dtc v-mid\\">
    Loading...
  </h1>
</div>"
`;
